spec/v1/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## v1.1
+
+-   Clarified that subjects are assumed to be immuatble and that it is
+acceptable to use a non-cryptographic digest (though cryptographic
+digests are still preferred).
```

Comment:
Nit: wonder if this should match the language in the spec a bit more closely.
```suggestion
digests are still strongly recommended).
```

Reply:
Makes sense to me.